sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 - Renamed `ContentLength` in `FileItemDetails` to `FileSize`.
 
+### Other Changes and Improvements
+
+- Updated some samples.
+- Fixed a read consistency issue.
```

Comment:
Why do we have this across all package CLs? Was the issue present everywhere?

Reply:
Yes, blob, datalake and file share services.